releasenotes/notes/resolve-secrets-in-diagnose-datadog-connectivity-a4e7061c6e2b9d7d.yaml

```diff
@@ -0,0 +1,12 @@
+# Each section from every release note are combined when the
+# CHANGELOG.rst is rendered. So the text needs to be worded so that
+# it does not depend on any information only available in another
+# section. This may mean repeating some details, but each section
+# must be readable independently of the other.
+#
+# Each section note must be formatted as reStructuredText.
+---
+fixes:
+  - |
+    `diagnose datadog-connectivity` subcommand now loads and resolves secrets before
```

Comment:
```suggestion
    `diagnose datadog-connectivity` subcommand now loads and resolves secrets before.
```

Reply:
@Kaderinho / @hush-hush  I think the trailing period should be at the next line, not on this one